UCLALibrary/prl-harvester
# [SERV-568] Add test OAI-PMH data provider

To test:

1. Run all the DMP-defined containers:

    ```bash
    # Tested with Maven 3.8.5
    mvn clean package docker:build docker:run -Ddocker.showLogs
    ```

2. Scroll to the `--- Set definition ---` section of the output of `provider-initializer` (should turn up in a search of the output for the string `provider-initializer> --- Set definition ---`)
3. Verify the output, including that the image URLs in the `dc:identifier` fields resolve
